### PR TITLE
Support multiple api definitions yamls

### PR DESCRIPTION
**Pull request by manorlh**
As a user, I want to use the assertions for multiple yamls files.

added the apiDefinitionsPaths option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Using the plugin is easy. Simply point the plugin to your API definitions file p
 - Coverage report (can be printed to your terminal or exported to a json file)
 - Supports response format from `axios`, `superagent`, `supertest`, `request` and `light-my-request` (used by `fastify`)
 - Supports OpenAPI 3.0
+- Supports multiple yaml documents
```
Comment (kobik) [inline README.md:22]: Supports multiple definition files

**Review by kobik — CHANGES_REQUESTED** [review]
Please also add this change to CHANGELOG.md

File: lib/plugins/chai/schema-matcher.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
 const validators = require('../../validators');
 
 module.exports = (Assertion, options) => {
-  Assertion.addMethod('matchApiSchema', function addApiSchemaMethod(apiDefinitionsPath) {
+  Assertion.addMethod('matchApiSchema', function addApiSchemaMethod() {
```
Comment (kobik) [inline lib/plugins/chai/schema-matcher.js:4]: this would be a breaking change. i think you should revert that

Comment (manorlh) [reply lib/plugins/chai/schema-matcher.js:4]: done

**Comment by manorlh** [comment]
@kobik ready for rereview

**Review by kobik — APPROVED** [review]

**Review by kobik — APPROVED** [review]

**Pull request merged**